src/event-helper-listen.js

```diff
@@ -41,11 +43,34 @@ export function internalListenImplementation(element, eventType, listener,
       throw e;
     }
   };
+
+  // Test whether browser supports the passive option or not
+  let passiveSupported = false;
+  try {
+    const options = Object.defineProperty({}, 'passive', {
```

Comment:
The polyfill is more general if you ask for capture instead. That way this code doesn't have to change for further options. If people really need this test, they can export a function for feature detection.